tests/FileFormat/PdfStatisticsTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Tests\CubeTools\CubeCommonBundle\FileFormat;
+
+use CubeTools\CubeCommonBundle\FileFormat\PdfStatistics;
+use PHPUnit\Framework\TestCase;
+
+class PdfStatisticsTest extends TestCase
+{
+    protected $folderWithPdfsForTests = 'tests' . DIRECTORY_SEPARATOR . 'Resources' . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR;
+
+    /**
+     * @var \CubeTools\CubeCommonBundle\FileFormat\PdfStatistics
+     */
+    protected $testObject;
+
+    public function setUp()
+    {
+        $this->testObject = new PdfStatistics();
+    }
+
+    public function testGetFormatOfPages()
+    {
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'oneA4grayscale.pdf');
+        $this->assertEquals(array('A4'), $this->testObject->getFormatOfPages());
+
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'twoA4firstGrayscaleSecondColor.pdf');
+        $this->assertEquals(array('A4', 'A4'), $this->testObject->getFormatOfPages());
+    }
+
+    public function testGetPagesInColor()
+    {
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'oneA4grayscale.pdf');
+        $this->assertEquals(0, count($this->testObject->getPagesInColor()));
+
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'twoA4firstGrayscaleSecondColor.pdf');
+        $this->assertEquals(array(1), $this->testObject->getPagesInColor());
+    }
+
+    public function testGetNumberOfPages()
+    {
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'oneA4grayscale.pdf');
+        $this->assertEquals(1, $this->testObject->getNumberOfPages(), 'Wrong number of pages calculated.');
+
+        $this->testObject->setFilename($this->folderWithPdfsForTests . 'twoA4firstGrayscaleSecondColor.pdf');
+        $this->assertEquals(2, $this->testObject->getNumberOfPages(), 'Wrong number of pages calculated.');
+    }
+}
```

Comment:
Since `identifyPageSize()` is public, we *could* add some tests for it (somewhen later).

Reply:
Currently it is covered by testGetFormatOfPages . We could make identifyPageSize() protected, but thought that maybe someone want to use identifyPageSize() autonomically.

Reply:
We just check one execution branch (or two?) of the function, because the size is always found. And it is always size A4. This could be enhanced. (Somewhen, not urgent ...)